### DISCUSSION TITLE
控制台警告3.2.6 Globe.terrainExaggeration was deprecated

### DISCUSSION THREAD

**Discussion by hanzhichao110** [zouyaoji/vue-cesium#544] [Q&A] [answered]
用的是3.2.6版本 控制台警告 如何去掉这个警告呢
Globe.terrainExaggeration was deprecated in CesiumJS 1.113.  It will be removed in CesiumJS 1.116. Use Scene.verticalExaggeration instead.

**Comment by zouyaoji** [comment, marked as answer]
收到，下个版本，处理Cesium更新问题。
